include/openssl/evp.h

```diff
@@ -2131,7 +2131,7 @@ OSSL_DEPRECATEDIN_3_0 void EVP_PKEY_meth_get_public_check
 OSSL_DEPRECATEDIN_3_0 void EVP_PKEY_meth_get_param_check
     (const EVP_PKEY_METHOD *pmeth, int (**pcheck) (EVP_PKEY *pkey));
 OSSL_DEPRECATEDIN_3_0 void EVP_PKEY_meth_get_digest_custom
-    (EVP_PKEY_METHOD *pmeth,
+    (const EVP_PKEY_METHOD *pmeth,
```

Comment:
Are these constifications going to be a problem? Do we need a separate vote?

Reply:
It's an API change, so yes we should vote.

Reply:
#16128